plugin/secrets_service_account_key.go

```diff
@@ -175,25 +175,17 @@ func (b *backend) secretKeyRevoke(ctx context.Context, req *logical.Request, d *
 
 func (b *backend) getSecretKey(ctx context.Context, s logical.Storage, rs *RoleSet, keyType, keyAlgorithm string, ttl int) (*logical.Response, error) {
 	cfg, err := getConfig(ctx, s)
```

Comment:
This error isn't checked.

Reply:
Added back in. Looks like a mess up during the rebase/merge.